src/unix/linux_like/linux/gnu/b64/powerpc64/align.rs

```diff
@@ -4,4 +4,92 @@ s_no_extra_traits! {
     pub struct max_align_t {
         priv_: [i64; 4]
     }
+
+    #[allow(missing_debug_implementations)]
+    pub struct ucontext_t {
+        pub uc_flags: ::c_ulong,
+        pub uc_link: *mut ucontext_t,
+        pub uc_stack: ::stack_t,
+        pub uc_sigmask: ::sigset_t,
+        pub uc_mcontext: mcontext_t,
+    }
+
+    #[allow(missing_debug_implementations)]
+    pub struct pt_regs {
```

Comment:
Which `pt_regs` does this use? It doesn't look like the generic one at https://github.com/torvalds/linux/blob/f43b15692129904ccc064180fa2dd796ba3843a5/arch/arc/include/asm/ptrace.h#L65C3-L65C15